src/renderers/webgl/WebGLState.js

```diff
@@ -751,9 +751,7 @@ function WebGLState( gl, extensions, capabilities ) {
 
 	// texture
 
-	function activeTexture( webglSlot ) {
-
-		if ( webglSlot === undefined ) webglSlot = gl.TEXTURE0 + maxTextures - 1;
+	function activeTexture( webglSlot = gl.TEXTURE0 + maxTextures - 1 ) {
```

Comment:
I'm not in favor of this kind of default parameters. Better to leave in a separate line.